tests/orgsGetByTag.test.js

```diff
@@ -4,6 +4,40 @@ var test = require('tape');
 var init = require('../example/server.js');
 var config = require('../config/load-config.js');
 
+test('Get all active and inactive organisations', function (t) {
+  init(config, function (err, server, pool) {
+    if (err) {
+      console.log('error initialise server', err);
+      return t.fail();
+    }
+
+    server.inject({
+      method: 'GET',
+      url: '/orgsGetByTag'
+    }, function (res) {
+      var expected = [
+        'Apple AAAA',
+        'Asda',
+        'Charcoal',
+        'Coca Cola',
+        'Co-op Group',
+        'dwyl',
+        'EMF',
+        'Fanta',
+      ];
+      expected.forEach(function (org, index) {
+        var regexp = new RegExp(org, 'g');
+        t.ok(res.payload.match(regexp), 'org name ' + org + ' displays as expected');
```

Comment:
Yeah I originally wanted to check that these org names would display, but also that they would only display once. I decided against it once I realised that that might not actually be the case but kept it as regex anyway.